package.json

```diff
@@ -31,5 +32,8 @@
     "./apps/*",
     "./packages/*"
   ],
-  "packageManager": "yarn@3.2.3"
+  "dependencies": {
```

Comment:
Should these be `dependencies` or `devDependencies`? I guess it doesn't make any real difference but I'm curious to know what are the recommendations here.